README.md

```diff
@@ -284,7 +108,7 @@ Authors
 
 Main author: Prabhu Ramachandran
 
-I gratefully acknowledge help from the following:
+Help from the following is gratefully acknowledged:
```

Comment:
Remove and replace with FOSSEE developers.  The contact section can simply be a standard fossee email address.